dynamic_analysis_pkg/dynamic_analysis_pkg/__init__.py

```diff
@@ -0,0 +1,167 @@
+
+from os import makedirs
+from os.path import join, exists, isdir, basename
+from shutil import move
+from datetime import datetime
+from tabulate import tabulate
+import pyunicore.client as unicore_client
+import pandas as pd
+
+UNICORE_ENDPOINT= 'https://bspsa.cineca.it/advanced/pizdaint/rest/core'
+
+
+class Results:
+
+    DEFAULT_CIRCUIT_DIR = '/home/data-bbp/20191017/'
+    DEFAULT_WD_BASE = '/home/simulation-results';
+    ORIGIN_CIRCUIT_PATH = '/store/hbp/ich002/antonel/O1/20191017/'
+    ORIGIN_DIR_PATH = '/scratch/snx3000/unicore/FILESPACE/'
+    job_base = join(UNICORE_ENDPOINT, 'jobs')
+    files_list = None
+    circuit_dir = None
+    local_dir = None
+    blueconfig = None
+
+    def __init__(self, token, sim_id, base_working_directory=None, new_circuit_path=None):
+        if base_working_directory == None:
+            base_working_directory = self.DEFAULT_WD_BASE
+        if new_circuit_path == None:
+            new_circuit_path = self.DEFAULT_CIRCUIT_DIR
+
+        self.define_paths(sim_id, base_working_directory, new_circuit_path)
+        self.fetch_results(token, sim_id)
+        self.blueconfig = join(self.local_dir, 'BlueConfig')
+
+    def fetch_results(self, token, sim_id=None):
+        if not sim_id:
+            print('[Error] Simulation ID not provided')
+            return
+        self.retrieve_sim_info(token, sim_id)
+        self.get_sim_results()
+
+    def retrieve_sim_info(self, token, sim_id=None):
+        tr = unicore_client.Transport(token)
+        job_url = join(self.job_base, sim_id)
+        job = unicore_client.Job(tr, job_url)
+        print('Fetching results of [{}]. Please wait ...'.format(job.properties['name']))
+        storage = job.working_dir
+        self.files_list = storage.listdir()
+
+    def get_sim_results(self):
+        self.download_blueconfig()
+        self.download_report()
+        self.download_out_dat()
+        print('Result were saved at: {}'.format(self.local_dir))
+
+    def define_paths(self, sim_id, working_directory, new_circuit_path):
+        self.local_dir = join(working_directory, sim_id)
+        if not exists(self.local_dir):
+            makedirs(self.local_dir)
+
+        if not isdir(new_circuit_path):
+            print('[ERROR] Circuit path does not exists')
```

Comment:
```suggestion
            L.error('[ERROR] Circuit path does not exists')
```
Shouldn't you raise here ?